.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
I don't think these new files (`.vs/*`) should be in repo.

Reply:
I'm sorry, I had no idea how to remove them.